### DISCUSSION TITLE
Streaming SSR

### DISCUSSION THREAD

**Discussion by theguriev** [nuxt/nuxt#20560] [Questions] [answered]
Hey guys

Do we have something similar to this in Nuxt? 
https://nextjs.org/docs/advanced-features/react-18/streaming
https://www.youtube.com/watch?v=yVOI81GKZBo

This feature looks like something very sexy.

**Comment by TheAlexLichter** [comment, marked as answer]
Not yet - follow https://github.com/nuxt/nuxt/issues/4753 along

**Reply by theguriev** [reply]
Thank you for the answer!
